nextjs-13/src/components/AppLayout/index.tsx

```diff
@@ -1,10 +1,48 @@
+'use client';
+
+import classNames from 'classnames';
+import Link from 'next/link';
+import { redirect, usePathname } from 'next/navigation';
+import { useSession } from 'next-auth/react';
+
 import Header from '@components/Header';
 
 export default function Layout({ children }: { children: React.ReactNode }) {
```

Comment:
You might want to name it as per the file name to remove any confusion:

```suggestion
export default function AppLayout({ children }: { children: React.ReactNode }) {
```

Ditto below https://github.com/nimblehq/js-framework-benchmark/pull/123/files#diff-b894ae1556a3dc5ef73e274f940f27c8ec2194179fb627726ab078fab9f6ebe3L3

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/commits/3b1a083a4265078a2f9995befb15bf3176a4d851)